src/core/tsi/ssl_transport_security.c

```diff
@@ -44,6 +44,9 @@
 
 #include <openssl/bio.h>
 #include <openssl/err.h>
+#if defined(OPENSSL_IS_BORINGSSL)
```

Comment:
No, the boringssl version has a stripped down version of crypto.h which doesn't include mem.h or define OPENSSL_free